src/System.Linq/src/System/Linq/Iterator.cs

```diff
@@ -9,58 +9,93 @@ namespace System.Linq
 {
     public static partial class Enumerable
     {
+        /// <summary>
+        /// A base class for enumerables that are loaded on-demand.
+        /// </summary>
+        /// <typeparam name="TSource">The type of each item to yield.</typeparam>
+        /// <remarks>
+        /// - The value of an iterator is immutable; when re-enumerated, it should always yield
```

Comment:
The really nice one that isn't on that page is `<inheritdoc/>`. Unfortunately the reason it isn't is that most of the IDEs (including VisualStudio) don't pay attention to it for intellisense, but SandCastle does so it's handy as far as generated documentation goes.